src/assets/scss/mts/modules/form/partials/_overrides.scss

```diff
@@ -8,15 +8,22 @@
 }
 
 .password-field {
-  -webkit-text-security: disc; 
+  caret-color: black;
+  -webkit-text-security: disc;
```

Comment:
The new font means we could remove this vendor prefix, as it looks very slightly, but not significantly different to the font implementation.
If we keep it in,  are we happy with the difference?
`caret-color` also is not supported in IE; does the colour fall back appropriately in IE?

Reply:
Yes, the caret looks fine in IE. I also agree with removing -webkit-text-security